personalities/sctfe/config.go

```diff
@@ -174,13 +182,13 @@ func ValidateLogConfig(cfg *configpb.LogConfig, origin string, projectID string,
 	}
 	vCfg.PrivKey = privKey
 
-	if cfg.RejectExpired && cfg.RejectUnexpired {
+	if rejectExpired && rejectUnexpired {
 		return nil, errors.New("rejecting all certificates")
 	}
 
 	// Validate the extended key usages list.
-	if len(cfg.ExtKeyUsages) > 0 {
-		for _, kuStr := range cfg.ExtKeyUsages {
+	if len(lExtKeyUsages) > 0 {
```

Comment:
Could the `if` be removed? (I think the `for`  will just not do anything if `lExtKeyUsages` is `[]`)

Reply:
Yes indeed!